sycl/test-e2e/DeviceArchitecture/device_architecture_on_host.cpp

```diff
@@ -0,0 +1,20 @@
+// UNSUPPORTED: cpu, cuda, hip, esimd_emulator
```

Comment:
are you adding support for OCL? I see only changes in the PR for UR and L0.

Reply:
@jandres742 see my comment https://github.com/intel/llvm/pull/9843#discussion_r1228241355